artifacts/definitions/Windows/EventLogs/Cleared.yaml

```diff
@@ -0,0 +1,107 @@
+  - name: SearchVSS
+    description: "Add VSS into query."
+    type: bool
+  - name: DateAfter
+    type: timestamp
+    description: "search for events after this date. YYYY-MM-DDTmm:hh:ssZ"
+  - name: DateBefore
+    type: timestamp
+    description: "search for events before this date. YYYY-MM-DDTmm:hh:ssZ"
+    
+sources:
+  - queries:
+      # Date bounds for time box
+      - LET DateAfterTime <= if(condition=DateAfter, 
+            then=timestamp(epoch=DateAfter), else=timestamp(epoch="1600-01-01"))
+      - LET DateBeforeTime <= if(condition=DateBefore, 
+            then=timestamp(epoch=DateBefore), else=timestamp(epoch="2200-01-01"))
+            
+      # Extract all target paths from specified globs
+      - LET evtxglobs <= SELECT expand(path=Glob) as EvtxGlob
+                     FROM parse_csv(filename=EvtxLookupTable, accessor='data')
+
+      - LET files = SELECT * FROM foreach(
+            row=evtxglobs,
+            query={
+                SELECT * FROM if(condition=SearchVSS,
+                    then= {
+                        SELECT * 
+                        FROM Artifact.Windows.Search.VSS(SearchFilesGlob=EvtxGlob)
+                    },
+                    else= {
+                        SELECT *
+                        FROM glob(globs=EvtxGlob)
+                    })
+                })
+
+     # Parse all target files, order by source and add dedupe string
+      - LET hits = SELECT *
+            FROM foreach(
+                row=files,
+                query={
+                    SELECT 
+                        timestamp(epoch=int(int=System.TimeCreated.SystemTime)) AS EventTime,
+                        System.Computer as Computer, 
+                        System.EventID.Value as EventID,
+                        System.EventRecordID as EventRecordID,
+                        if(condition= System.EventID.Value = 1102,
+                            then= System.Channel,
+                            else= UserData.LogFileCleared.Channel) as Channel,
+                        if(condition= System.EventID.Value = 1102, 
+                            then= UserData.LogFileCleared.SubjectDomainName + '\\' +
+                                UserData.LogFileCleared.SubjectUserName,
+                            else= UserData.LogFileCleared.SubjectDomainName + '\\' +
+                                UserData.LogFileCleared.SubjectUserName) as UserName,
+                        if(condition= System.EventID.Value = 1102,        
+                            then= UserData.LogFileCleared.SubjectUserSid,
+                            else= System.Security.UserID) as SecurityID,    
+                        Message,
+                        if(condition=Source, then=Source, else=FullPath) as Source,
+                        format(format="%v-%v-%v",args=[System.EventID.Value,System.EventRecordID,
+                            timestamp(epoch=int(int=System.TimeCreated.SystemTime))]) as _Group
```

Comment:
do you need the extra int here?

Reply:
fixed and pushed.

Reply:
@scudette I just thought, actually the int for the EventTime field is preferred due to it making the time a bit more friendly:
2020-06-08T05:58:09Z
vs
2020-06-08T05:58:09.212196826Z